harness/tests/cli/test_experiment.py

```diff
@@ -7,8 +7,9 @@
 import determined
 import determined.cli
 from determined.common import constants
-from determined.common.api import bindings, certs
+from determined.common.api import _session, bindings, certs
 from determined.common.api.authentication import Authentication
+from determined.common.experimental import experiment
```

Comment:
this isn't self-evident, but the original python sdk didn't follow the code layout rfc.  This path is deprecated, and you should import like this instead:

```python
from determined.experimental import client

...
client.ExperimentReference
```

Reply:
I'm all for that deprecation, and I've made the change in the code.

I haven't tested it, but maybe we could stick a `DeprecationWarning` into `determined.common.experimental.experiment` encouraging the new path and then in `determined.experimental.client` stick the import inside a `with warnings.catch_warnings()`?

```python
with warnings.catch_warnings():
  from determined.common.experimental.experiment import (  # noqa: F401
    ExperimentReference,
    ExperimentState,
)
```
...etc...

Good idea?

Reply:
https://determinedai.atlassian.net/browse/MLG-381